content/classes/reading/modifiers/index.md

```diff
@@ -0,0 +1,110 @@
+We should avoid giving everything default access when creating classes in Java and instead think carefully about what level of access each field and method should have.
+
+The table below details whether or not information can be accessed at different levels based on the access modifier. For example, a field with the `private` access modifier can be accessed within the class, but cannot be accessed outside the class at the world-level. In Java, world-level is the level of the whole application and contains all of the packages and classes. While we will discuss later how to decide which access modifier to use for different scenarios, you should save this table now as reference for those conversations.
+
+| Modifier   | Class | Package | World |
+|------------|-------|---------|-------|
+| `public`   | Yes   | Yes     | Yes   |
+| `protected`| Yes   | Yes     | No    |
+| (no modifier, aka *package-private* or *default*) | Yes | Yes | No |
+| `private`  | Yes   | No      | No    |
+
+{{% notice blue Note "rocket" %}}
+If you would like to learn more about access modifiers, you should check out the [Oracle documentation](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html) on the subject.
+{{% /notice %}}
+
+Let's take a look at our `HelloWorld` class again and add some access
+modifiers.
+
+{{% notice blue Example "rocket" %}}
+```java
+public class HelloWorld {
+
+   private String message = "Hello World";
+
+   public void sayHello() {
+      System.out.println(message);
+   }
+
+}
+```
+{{% /notice %}}
+
+Since `message` only needs to be used by `sayHello`, we declare it to be `private`. Since we want `sayHello` to be usable by anybody else, we declare it to be `public`.
+
+{{% notice blue Note "rocket" %}}
+In Java, you should always use the most restrictive access modifier possible. Minimizing access to class members allows code to be refactored more easily in the future, and hides details of how you implement your classes from others.
+
+This makes your code more modular and modifiable. Each public member that you expose is another field or property that can be referenced directly elsewhere in any program using your class. Thus, changing any such field in your code could potentially break any code referencing such members. The fewer public members, the more you can change your code without breaking stuff elsewhere.
+{{% /notice %}}
+
+{{% notice blue Note "rocket" %}}
+If you want to learn more about controlling access to members of a class, check out this information from [Oracle](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html).
+{{% /notice %}}
+
+### Check Your Understanding
+
+{{% notice blue Question "rocket" %}}
+For this question, refer to the code block below.
+
+```java
+public class Greeting {
+
+   String name = "Jess";
+
+   public void sayHello() {
+      System.out.println("Hello " + this.name + "!");
+   }
+}
+```
+
+What access modifier would you give `name`?
+
+a. no access modifier
```

Comment:
<img width="822" alt="CYU Modifiers" src="https://github.com/LaunchCodeEducation/java-web-dev-curriculum/assets/39868164/8aa577da-c7cd-4187-919e-91175e921522">
 
Fix formatting so each choice is on a separate line.

Reply:
fixed